driver/bpf/fillers.h

```diff
@@ -6725,4 +6725,33 @@ FILLER(sys_prctl_x, true)
 	return res;
 }
 
+	FILLER(sys_memfd_create_x,true)
+	{
+		int retval;
+		unsigned long val;
+		unsigned long flags;
+		unsigned long res;
+
+		/*
+		* fd
+		*/
+		retval = bpf_syscall_get_retval(data->ctx);
+		res = bpf_push_s64_to_ring(data, retval);
+		CHECK_RES(res);
+
+		/*
+		* name
+		*/
+		val = bpf_syscall_get_argument(data, 0);
+		res = bpf_push_u64_to_ring(data, val);
+		CHECK_RES(res);
+
+		/*
+		* flags
+		*/
+		val = bpf_syscall_get_argument(data, 2);
```

Comment:
According to `int memfd_create(const char *name, unsigned int flags);` prototype, the argument number should be `1` instead of `2`.
```suggestion
		val = bpf_syscall_get_argument(data, 1);
```